services/static-webserver/client/source/class/osparc/desktop/preferences/pages/GeneralPage.js

```diff
@@ -142,6 +141,21 @@ qx.Class.define("osparc.desktop.preferences.pages.GeneralPage", {
       jobConcurrencySpinner.addListener("changeValue", e => this.self().patchPreference("jobConcurrencyLimit", jobConcurrencySpinner, e.getData()));
       form.add(jobConcurrencySpinner, this.tr("Maximum concurrent jobs"));
       box.add(new qx.ui.form.renderer.Single(form));
+      return box;
+    },
+    __createUserPrivacySettings: function() {
+      const box = this._createSectionBox("Privacy settings");
+
+      const label = this._createHelpLabel(this.tr("Help improve user experience."));
```

Comment:
NOTE: double check the wording of this with @eofli @mrnicegyu11  and other people

THOUGHT: I have been encouraging to export all texts so that they can control messages etc. The front-end already supports that, This way they could change all labels without us having to change thte code!

Reply:
I would leave such an export change up to the frontend people. Since the entire section would require that. @ignapas @jsaq007 
I think such a change should be done at a later date.